plugin/audit_log/audit_log.c

```diff
@@ -242,15 +242,37 @@ void csv_escape(const char *in, size_t *inlen, char *out, size_t *outlen)
   escape_buf(in, inlen, out, outlen, rules);
 }
 
+/*
+  Escape string according to audit_log_format.
+
+  @param[in]  in           Input string
+  @param[in]  inlen        Length of the input string
+  @param[in]  out          Output buffer
+  @param[in]  outlen       Length of the output buffer
+  @param[out] endptr       A pointer to the character after the
+                           last escaped character in the output
+                           buffer
+  @param[out] full_outlen  Full lenght of the output buffer needed to
```

Comment:
I'd say something like "Length of the output buffer that would be needed to store complete non-truncated escaped input buffer"